### DISCUSSION TITLE
add more build targets

### DISCUSSION THREAD

**Discussion by AdamJel** [massCodeIO/massCode#147] [Q&A]
Hello,

could you please provide more variants within releases? There is a substantial amount of people, who literally hate snap and don't have it on their system. Current situation is such, that everybody who doesn't have snap must build it themself. Perhaps a packages for basic distributions?

Thank you

**Comment by antonreshetov** [comment]
https://github.com/massCodeIO/massCode/discussions/87

**Comment by antonreshetov** [comment]
I have no problem adding other build targets for Linux, the problem is that I cannot test them. Consequently, there will be issues of bugs that I will not even be able to reproduce to solve them.
